core/src/main/java/io/kestra/core/queues/QueueFactoryInterface.java

```diff
@@ -16,11 +17,11 @@ public interface QueueFactoryInterface {
     String FLOW_NAMED = "flowQueue";
     String TEMPLATE_NAMED = "templateQueue";
     String WORKERTASKLOG_NAMED = "workerTaskLogQueue";
+    String WORKERTASKMETRIC_NAMED = "workerTaskMetricQueue";
```

Comment:
```suggestion
    String METRIC_NAMED = "workerTaskMetricQueue";
```

Reply:
I choose this name to be as close as possible to the name of the log queue, and to differentiate with Kestra metrics

Reply:
OK, done